@theme/components/Amendments.tsx

```diff
@@ -0,0 +1,391 @@
+import * as React from 'react';
+import { Link } from '@redocly/theme/components/Link/Link';
+import { useThemeHooks } from '@redocly/theme/core/hooks';
+
+type Amendment = {
+  name: string;
+  rippled_version: string;
+  tx_hash?: string;
+  consensus?: string;
+  date?: string;
+  id: string;
+  eta?: string;
+};
+
+type AmendmentsResponse = {
+  amendments: Amendment[];
+};
+
+type AmendmentsCachePayload = {
+  timestamp: number;
+  amendments: Amendment[];
+}
+
+// API data caching
+const amendmentsEndpoint = 'https://vhs.prod.ripplex.io/v1/network/amendments/vote/main/';
+const amendmentsCacheKey = 'xrpl.amendments.mainnet.cache';
+const amendmentsTTL = 12 * 60 * 60 * 1000; // 12 hours in milliseconds
+
+function readAmendmentsCache(): Amendment[] | null {
+  if (typeof window === 'undefined') return null;
+  try {
+    const raw = window.localStorage.getItem(amendmentsCacheKey);
+    if (!raw) return null;
+    const parsed: AmendmentsCachePayload = JSON.parse(raw);
+    if (!parsed || !Array.isArray(parsed.amendments)) return null;
+    const fresh = Date.now() - parsed.timestamp < amendmentsTTL;
+    return fresh ? parsed.amendments : null;
+  } catch {
+    return null;
+  }
+}
+
+function writeAmendmentsCache(amendments: Amendment[]) {
+  if (typeof window === 'undefined') return;
+  try {
+    const payload: AmendmentsCachePayload = { timestamp: Date.now(), amendments };
+    window.localStorage.setItem(amendmentsCacheKey, JSON.stringify(payload));
+  } catch {
+    // Ignore quota or serialization errors
+  }
+}
+
+// Sort amendments table
+function sortAmendments(list: Amendment[]): Amendment[] {
+  const getStatusPriority = (amendment: Amendment): number => {
+    if (amendment.eta) return 0;          // Expected
+    if (amendment.consensus) return 1;    // Open for Voting
```

Comment:
I would think "Expected" should appear _between_ "Open for Voting" and "Enabled" because that's how the states transition in the happy path: Voting → Expected → Enabled.

```suggestion
    if (amendment.consensus) return 0;    // Open for Voting
    if (amendment.eta) return 1;          // Expected
```

Reply:
Good point, I'll make the change.